modules/end-user-guide/partials/proc_creating-a-workspace-from-template-devfile.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+
+[id="creating-a-workspace-from-template-devfile_{context}"]
+= Creating a workspace from template devfile
```

Comment:
from a devfile template or from a template devfile, but definitely an "a" is missing.